### PR TITLE
add queue to save config while switching teams

### PR DESCRIPTION
**Pull request by joecohens**
This fixes an issue by switching teams very fast, they get jumpy because of the sync with the file system and the refresh. What do you think @leo is this too crazy lol?

Jumpy
https://d.pr/i/kMkjan

Not Jumpy
https://d.pr/i/p21gCZ

### CODE REVIEW

File: renderer/components/feed/switcher.js [hunk 1]
```diff
@@ -469,6 +470,10 @@ class Switcher extends Component {
       this.updateTouchBar()
     }
 
+    while (this.state.queue.length > 0) {
+      this.state.queue.shift()()
```
Comment (pranaygp) [inline renderer/components/feed/switcher.js]: I think you'll need to explicitly `this.setState` over here

Comment (joecohens) [reply renderer/components/feed/switcher.js]: Whoops you'r completely right, thanks! 😉

**Comment by pranaygp** [comment]
You *might* need to rebase and run on canary to fix those tests. I *just* fixed those tests in 888794f72389d8ca022577061ab46da1f49674a2

**Comment by joecohens** [comment]
🙌

**Comment by pranaygp** [comment]
Thanks @joecohens 

I'm including this in the new canary release

**Comment by joecohens** [comment]
🍻

**Pull request merged**